templates/base.html

```diff
@@ -1,40 +1,37 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-  {% block title %}<title>Taxi Service</title>{% endblock %}
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
-      integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
-      crossorigin="anonymous">
-  <!-- Add additional CSS in static file -->
-  {% load static %}
-  <link rel="stylesheet" href="{% static 'css/styles.css' %}">
-</head>
-
-<body>
-<div class="container-fluid">
-  <div class="row">
-    <div class="col-sm-2">
-
-      {% block sidebar %}
-        {% include "includes/sidebar.html" %}
-      {% endblock %}
-
+  <head>
+    {% block title %}
+      <title>Taxi Service</title>
+    {% endblock %}
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous" />
+    <!-- Add additional CSS in static file -->
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}" />
+  </head>
+
+  <body>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-sm-2">
+          {% block sidebar %}
+            {% include 'includes/sidebar.html' %}
+          {% endblock %}
+        </div>
+        <div class="col-sm-10">
+          <div class="content-block">
+            <!-- block content -->
+            {% block content %}
+            {% endblock %}
+          </div>
+
+          {% block pagination %}
+            {% include 'includes/pagination.html' %}
```

Comment:
Ensure that the `includes/pagination.html` template exists and is correctly implemented, as it is included in the pagination block.